docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Managing tracking scenarios in the console"
+sidebar_label: "🆕 Managing tracking scenarios in the console"
+sidebar_position: 80
+sidebar_custom_props:
+  offerings:
+    - enterprise
+---
+
+
+## Creating a New Tracking Scenario using Console
+
+Tracking Scenarios are directly linked to Data Structures and represent the events that are documented and tracked using a certain Data Structure.
+
+To create a new Tracking Scenario, navigate to the **Data Structures** option in the menu. From there, select a data structure from the list.
```

Comment:
I would make that a link to “managing data structures via UI”

Reply:
Done